src/stdatamodels/jwst/datamodels/schemas/outlierpars.schema.yaml

```
@@ -61,3 +61,5 @@ allOf:
         datatype: float32
       - name: kernel_size
         datatype: [ascii, 10]
+      - name: ifu_second_check
+        type: boolean
```

```suggestion
        datatype: bool8
```

Maybe give this a try.